influxdb3_load_generator/analysis/templates/index.html

```diff
@@ -0,0 +1,236 @@
+        configName2Select.innerHTML = '';
+
+        fetch(`/api/config-names?test_name=${testName}`)
+            .then(response => response.json())
+            .then(configNames => {
+                for (const configName in configNames) {
+                    const runTimes = configNames[configName];
+                    runTimes.forEach(runTime => {
+                        const option1 = document.createElement('option');
+                        option1.value = `${configName}/${runTime}`;
+                        option1.textContent = `${configName}/${runTime}`;
+                        configName1Select.appendChild(option1);
+
+                        const option2 = document.createElement('option');
+                        option2.value = `${configName}/${runTime}`;
+                        option2.textContent = `${configName}/${runTime}`;
+                        configName2Select.appendChild(option2);
+                    });
+                }
+            });
+    }
+
+    // Fetch aggregated data and render graphs when the compare button is clicked
+    compareBtn.addEventListener('click', () => {
+        const selectedTestName = testNameSelect.value;
+        const [selectedConfigName1, selectedRunTime1] = configName1Select.value.split('/');
+        const [selectedConfigName2, selectedRunTime2] = configName2Select.value.split('/');
+
+        Promise.all([
+            fetch(`/api/aggregated-data?test_name=${selectedTestName}&config_name=${selectedConfigName1}&run_time=${selectedRunTime1}`),
+            fetch(`/api/aggregated-data?test_name=${selectedTestName}&config_name=${selectedConfigName2}&run_time=${selectedRunTime2}`)
+        ])
+            .then(responses => Promise.all(responses.map(response => response.json())))
+            .then(data => {
+                const config1Data = data[0];
+                const config2Data = data[1];
+
+                renderGraph('lines-per-second-chart', 'Lines per Second', config1Data.write_data, config2Data.write_data, 'lines', 10000);
+                renderGraph('write-latency-chart', 'Write Latency (ms)', config1Data.write_data, config2Data.write_data, 'latency', 10000, 'median');
+                renderGraph('queries-per-second-chart', 'Queries per Second', config1Data.query_data, config2Data.query_data, 'lines', 10000);
+                renderGraph('query-latency-chart', 'Query Latency (ms)', config1Data.query_data, config2Data.query_data, 'latency', 10000, 'median');
+                renderGraph('cpu-usage-chart', 'CPU Usage (%)', config1Data.system_data, config2Data.system_data, 'cpu_usage');
+                renderGraph('memory-usage-chart', 'Memory Usage (MB)', config1Data.system_data, config2Data.system_data, 'memory_bytes');
+            });
+    });
+
+    // Render a graph using Chart.js
+    // Render a graph using Chart.js
+    function renderGraph(chartId, title, config1Data, config2Data, yAxisKey, interval = 10000, aggregateFunction = 'sum') {
```

Comment:
It might be helpful to rename `interval` to express the units it is in, I believe it is milliseconds, so `interval_ms`